### PR TITLE
{compiler}[intel/2017b] LLVM v5.0.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2148.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/11712351e86f35954a803ac268daee4c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2659.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/fd01abf32e54b22b0ff555e8742899c8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
node2077.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4c98fe3cd9315ecbf84f6ac2c3092b40 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 25 out of 25 (7 easyconfigs in this PR)
node2702.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c189d2e005aae4caae25ee486ea69ef8 for a full test report.

File: easybuild/easyconfigs/l/LLVM/LLVM-3.9.0-foss-2016b.eb [hunk 1]
```diff
@@ -30,7 +30,7 @@ dependencies = [
 
 configopts = '-DBUILD_SHARED_LIBS=ON '
 # required to install extra tools in bin/
-configopts += '-DLLVM_INSTALL_UTILS=ON -DLLVM_TARGETS_TO_BUILD=X86 -DLLVM_ENABLE_ZLIB=ON '
+configopts += "-DLLVM_INSTALL_UTILS=ON -DLLVM_TARGETS_TO_BUILD=X86 -DLLVM_ENABLE_ZLIB=ON -DLLVM_ENABLE_RTTI=1 "
```
Comment (wpoely86) [inline easybuild/easyconfigs/l/LLVM/LLVM-3.9.0-foss-2016b.eb]: if you change the quotes on this line, also do it on the others

Comment (boegel) [reply easybuild/easyconfigs/l/LLVM/LLVM-3.9.0-foss-2016b.eb]: I did where I could, but in some places there's a technical reason to use outer single quotes (which I prefer not to use when I can, for values that include spaces).

**Pull request merged**